contrib/win32/win32compat/ssh-agent/agent.c

```diff
@@ -289,6 +294,13 @@ get_con_client_info(struct agent_connection* con)
 		goto done;
 	}
 
+	// Get username
+	sshagent_con_username= get_username(info->User.Sid);
+	if (sshagent_con_username)
+		debug_f("sshagent_con_username: %s", sshagent_con_username);
+	else
+		error_f("Failed to get the userName");
```

Comment:
Do we want to exit() here?

Reply:
no.. Username is used only for the FIDO scenario. In case if we fail to get a username, I don't want to break other scenarios which don't require the username.